incubator/hnc/hack/krew-kubectl-hns.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hns
+spec:
+  shortDescription: Manipulate hierarchical namespaces provided by HNC.
```

Comment:
something as short as:

> Manage hierarchical namespaces

can be better.

Reply:
sg - I want to keep the term "HNC" in there somewhere but I'll shorten the rest.

Reply:
Shortened to "Manage hierarchical namespaces (part of HNC)"